Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/GyrometerSamplePage.xaml

```diff
@@ -0,0 +1,81 @@
+﻿<Page x:Class="Uno.Gallery.Views.Samples.GyrometerSamplePage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  xmlns:smtx="using:ShowMeTheXAML"
+	  xmlns:local="using:Uno.Gallery"
+	  xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+	  mc:Ignorable="d">
+
+	<Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+		<local:SamplePageLayout IsDesignAgnostic="True">
+			<local:SamplePageLayout.DesignAgnosticTemplate>
+				<DataTemplate>
+					<StackPanel>
+						<smtx:XamlDisplay UniqueKey="GyrometerSamplePage_Sample"
+										  smtx:XamlDisplayExtensions.IgnorePath="XX"
+										  smtx:XamlDisplayExtensions.Header="Gyrometer">
+							<Grid>
+								<Grid.RowDefinitions>
+									<RowDefinition Height="auto" />
+									<RowDefinition Height="auto" />
+								</Grid.RowDefinitions>
+								<!-- C# code
+var gyrometer = Gyrometer.GetDefault();
+gyrometer.ReadingChanged += Gyrometer_ReadingChanged;
+
+private async void Gyrometer_ReadingChanged(Gyrometer sender, GyrometerReadingChangedEventArgs args)
+{
+    // If you want to update the UI in some way, ensure the Dispatcher is used,
+    // as the ReadingChanged event handler does not run on the UI thread.
```

Comment:
it is not that ReadingChanged doesnt work outside of dispatcher,
but updating the ui (changing those 4 properties that are data-bound) needs to be done on dispatcher